wesnoth-units.po

```diff
@@ -3549,11 +3573,26 @@ msgid ""
 "‘reflection pools’ and steer them is one of her greatest — and most feared — "
 "abilities."
 msgstr ""
-
+"Vědmy jako jsou sylfy jsou nesmírně silné těmi nejnepředstavitelnějšími "
+"způsoby a jsou mistryněmi manipulace spojení mezi pozemským a tajemným "
+"světem. Dlouhá léta strávená nahlížení do éterické říše narušila schopnost "
+"těchto elfek nahlížet fyzický svět. Na oplátku je jim poskytnut abstraktní "
+"náhled, z kterého získávají schopnost vidět jeden nebo dokonce několik "
```

Comment:
```suggestion
"pohled, ze kterého získávají schopnost vidět jeden nebo dokonce několik "
```

Reply:
No ideální to také není, ten pohled se mi zdá malinko hoří. Ale bylo by tam hodně náhledů. Ještě by tam mohlo být nazírání, ale už se mi nějak nechte se v tom hrabat.